02-getting_started.Rmd

```diff
@@ -1,67 +1,249 @@
 
-## How to download the R sources directly or from the svn repository?
+## Building R 
+
+### Linux
+
+Here are the basic steps intended as a checklist.
+For complete instructions please see the section in [R-admin](https://cran.r-project.org/doc/manuals/r-devel/R-admin.html#Installing-R-under-Unix_002dalikes).
+
+1.  Download the latest recommended packages[^02-getting_started-2]:
+
+        $TOP_SRCDIR/tools/rsync-recommended
+
+3.  Create the build directory in the `BUILDDIR`:
+
+        export BUILDDIR="$HOME/bin/R"
+        mkdir -p $BUILDDIR
+        cd $BUILDDIR
+        
+4.  Configure the R installation (with `--enable-R-shlib` so that RStudio IDE can use it):
+        
+        $TOP_SRCDIR/configure --enable-R-shlib
+        
+4.  Build R :
+
+        make 
+
+
+5.  Check that R works as expected:
+
+        make check
+
+    There are other checks you can run:
+
+        make check-devel
+        make check-recommended
+
+
+[^02-getting_started-2]: Recommended packages are not in the subversion repository.
+
+
+If we don't want to build R in a different directory than the source code we can simply use:
+
+    cd $TOP_SRCDIR 
+    svn update
+    tools/rsync-recommended
+    $TOP_SRCDIR/configure  --enable-R-shlib
+    make 
+    make check
+
+Once you successfully build R from source you can modify R source code to fix an issue: Prepare a patch (See [this guide](https://www.r-project.org/bugs.html#how-to-submit-patches)) and after checking that R works as intended (`make check-devel`) submit the patch for the R core consideration.
+(See the [lifecycle of a patch](#FixBug) chapter).
+
+
+To use the r-devel version in RStudio to you can do the following:
+
+    export RSTUIDO_WHICH_R="$BUILDDIR/bin/R"
+    cd $TOP_SRCDIR
+    rstudio
+
+
+### Windows
+
+#### Binaries
+
+The latest documentation about installing R and packages from source at the current time can be find [here](https://cran.r-project.org/bin/windows/base/howto-R-4.2.html).
+
+1.  The binary builds of R for Windows can be downloaded and installed from [here](https://cran.r-project.org/bin/windows/base/).
+    Along with the link to the latest stable release, this page also contains links to the binary builds of `r-patched` and `r-devel`.
+
+2.  Click on the download links to download an executable installer.
+
+3.  Select the language while installing, read the GNU general public license information, and select the destination location to start the installation.
+    You will be prompted to select components at this stage: `User installation`, `64-bit User installation`, or `Custom installation`.
+    The default option may be chosen for the questions from this step onwards to complete the installation.
+    
+You can also use the [binaries for `r-devel` made available](https://cran.r-project.org/bin/windows/base/rdevel.html) which are not so up to date. 
```

Comment:
I would not be so negative here, they are daily builds, so may be in sync with Subversion or a small number of  commits behind, which will rarely affect people's work.